src/trivyHelper.ts

```diff
@@ -57,8 +59,12 @@ export async function runTrivy(): Promise<TrivyResult> {
 }
 
 export async function getTrivy(): Promise<string> {
-    const latestTrivyVersion = await getLatestTrivyVersion();
 
+    let latestTrivyVersion = inputHelper.trivyVersion;
+    if(trivyVersion == 'latest'){
+        latestTrivyVersion = await getLatestTrivyVersion();
+    }
```

Comment:
I think the imports are mixed up here (`inputHelper.trivyVersion` and `trivyVersion` are the same from different imports).
Suggestion:
```
let latestTrivyVersion = inputHelper.trivyVersion;
if (latestTrivyVersion === 'latest') {
  latestTrivyVersion = await getLatestTrivyVersion();
}
```

Reply:
Yes, thank you for that! I overlooked that one.